api/lib/infrastructure/repositories/training-repository.js

```diff
@@ -0,0 +1,16 @@
+const { knex } = require('../../../db/knex-database-connection');
+const Training = require('../../domain/models/Training');
+
+async function list() {
+  const trainings = await knex('trainings').orderBy('id', 'asc');
```

Comment:
À rediscuter fonctionnellement je pense mais pas bloquant

Reply:
Pourquoi à rediscuter ? C'est l'ordre actuel et comme ça fini dans le redis puis dans le in-memory cache, il n'y a à mon sens pas d'intêrêt de le trier ici. 

Je comprends l'intérêt de fixer l'ordre pour les tests cependant.

Reply:
On peut retrier côté endpoint LCMS ?

Pour l'instant on ne sait pas dans quel ordre ça sera affiché pour l'utilisateur, c'était ça mon point.

Reply:
L'endpoint LCMS /releases/:id et /releases/latest permettent juste de récupérer une release. 

La fonction `list` ici, permet juste de déterminer l'ordre dans la release (dans du json dans la table `releases`). 

Si il doit y avoir un ordre pour l'utilisateur ça ne se passera pas dans cette API, mais dans l'API de Pix.

Reply:
OK c'était pas clair pour moi.